### DISCUSSION TITLE
How to make documentation section as the home page

### DISCUSSION THREAD

**Discussion by Leroy-X** [imfing/hextra#68] [Q&A] [answered]
Hello, I would like to go directly to the document section to replace the default home page. What should I do?
So I added `contentDir: content/docs` in the configuration file.
But hextra seems to recognize `_index.md` in the docs directory as the main page. Therefore the left sidebar will not appear. Even if I use `menu:
   The sidebar: ` configuration doesn't seem to work either.
Looking forward to your help, this has been bothering me for a long time.
Thanks!!!

**Comment by imfing** [comment, marked as answer]
Please see this discussion:

https://github.com/imfing/hextra/discussions/18#discussioncomment-6879900

**Comment by CaseyLabs** [comment]
Solution for future searchers: 

How do you make the `content/Docs` folder the hompage in a Hextra site?

Workaround:

- Edit `content/_index.md`

- Add the `cascade` parameter to the top of the file like this:

```
---
title: My Site
toc: false
cascade:
  type: docs
---
```

Save and re-run `hugo server`. This will build the root index file with the Doc template, which will allow you to build out the rest of the site as if it was landing on `content/docs`

**Reply by Leroy-X** [reply]
Thank you very much, got it.
